GithubConnect/GithubConnect.psm1

```diff
@@ -1,3 +1,4 @@
-$Scriptroot = Split-Path $script:MyInvocation.MyCommand.Path
-
-Get-ChildItem $Scriptroot *.ps1 -Recurse | % { Import-Module $_.FullName }
+$Scriptroot = Split-Path $script:MyInvocation.MyCommand.Path
+
+Get-ChildItem $Scriptroot\Public *.ps1 -Recurse | ForEach-Object { Import-Module $_.FullName }
+. $Scriptroot\*.Tests.ps1 #This is only in still as this was how I found the Module running - Advise that we remove this line on v1 release

```

Comment:
Can you remove this line now maybe? Makes Import-Module look a bit strange. What were your issues running the module without that line?